### DISCUSSION TITLE
Bank plugin that tell you last time you used an item

### DISCUSSION THREAD

**Discussion by Zambookie** [runelite/runelite#16905] [Ideas]
I would love a plugin that tell me last time i used an item in my bank it will help loads with bank cleaning and organizing. I'm sure my bank has items i haven't touched in years but don't realize that i don't use them and can get rid of them.

**Comment by Zambookie** [comment]
what does should-be-external mean did i do something wrong?

**Reply by Felanbird** [reply]
The label is to indicate to someone who wanted to add this feature, that it should be added via the plugin-hub and not added to any of the main plugins.

**Reply by Zambookie** [reply]
oh ok thx

**Comment by Zambookie** [comment]
if it helps you can determine the age with colors that we can change the length of time for the color like green would be past 3 months yellow could be past 6 months and red could be 6 months + but have the option for someone to change it to weeks or whatever they like
